src/ParallelAlgorithms/Interpolation/InterpolationTargetDetail.hpp

```diff
@@ -522,13 +521,31 @@ auto block_logical_coords(const db::DataBox<DbTags>& box,
   }
 
   // Time-independent case.
-  return ::block_logical_coordinates(
-      domain, InterpolationTargetTag::compute_target_points::points(
-                  box, tmpl::type_<Metavariables>{}, temporal_id));
+  return ::block_logical_coordinates(domain, coords);
+}
+
+/// Version of block_logical_coords that computes the interpolation
+/// points on the fly.  This version is called when the interpolation
+/// points are (or might be) time-dependent in the
+/// InterpolationTargetTag's frame.
```

Comment:
I think it would be helpful to add to this comment that this is primarily used without an Interpolator parallel component. (Unless that's the other overload? I think some clarification on which is used when you have the Interpolator parallel component would be nice)